client/js/bundles/statement/bulkEditStatement.js

```diff
@@ -29,4 +29,8 @@ const components = {
   DpBulkEditStatement
 }
 
+if (hasPermission('area_admin_boilerplates')) {
+  stores.boilerplates = BoilerplatesStore
```

Comment:
Remove the store above, too.

Reply:
fixed in f38170fcbc35bb136a7f6770a32d31c05451a7d2